user-docs/pupil-player.md

```diff
@@ -198,17 +198,45 @@ These plugins are simple unique plugins, that operate on the gaze data for analy
-  + Surface name and size - In the `Marker Detector` GUI window, define the surface name and real world size. *Note* - defining size is important as it will affect how heatmaps are rendered.
+  + Surface name and size - In the `Marker Detector` GUI window, define the surface name and real world size. 
+  *Note* - defining size is important as it will affect how heatmaps are rendered.
   + Set trim marks - optional, but if you want to export data for a specific range, then you should set the trim marks.
-  + Recalculate gaze distributions - click the `(Re)calculate gaze distributions` button after specifying surface sizes. You should now see heatmaps in the Player window (if gaze positions were within your defined surfaces).
+  + Recalculate gaze distributions - click the `(Re)calculate gaze distributions` button after specifying surface sizes. 
+  You should now see heatmaps in the Player window (if gaze positions were within your defined surfaces).
   + Export gaze and surface data - click `e` and all surface metrics reports will be exported and saved for your trim section within your `export` folder.
 
+All files generated by the `Offline Surface Detector` will be located in the subfolder `surfaces`. 
+The different reported metrics are:
+
+  + `surface_visibility.csv` - Overview of how many world camera frames each surface was contained in.
+  + `surface_gaze_distribution.csv` - Overview of how many gaze samples have been collected on each individual surface and outside of surfaces.
+  + `surface_events.csv` - List of gaze-enter and gaze-exit events for all surfaces.
+
+Further the following metrics are reported for every individual surface. 
+Each surface has a name, which can be manually set as described above. 
+This name is augmented by an automatically generated numerical identifier.
+  
+  + `heatmap_<surface_name>_<surface_id>.png` - Heatmap of gaze positions on the surface aggregated over the entire export.
+  + `fixations_on_surface_<surface_name>_<surface_id>.csv` - A list of fixations that have occured on the surface. 
+  + `gaze_positions_on_surface_<surface_name>_<surface_id>.csv` - A list of gaze datums on the surface. 
+  The values include the gaze point in two different coordinates systems.
+  `x_norm` and `y_norm` are coordinates between 0 and 1, where `(0,0)` is the bottom left corner of the surface and `(1,1)` is the top right corner.
+  `x_scaled` and `y_scaled` contain the same coordinates but scaled with the size defined for the surface.
+  + `srf_positons_<surface_name>_<surface_id>` - List of surface positions in 3D. 
+  The position is given as the 3D pose of the surface in relation to the current position of the scene camera.
+  `m_to_screen` is a matrix transforming coordinates from the camera coordinate system to the surface coordinate system.
```

Comment:
I would like to change the naming here to `m_to_surface` and `m_from_surface`. The word screen is not correct in many use cases.

Reply:
fine with me.